app/src/pages/inside/projectSettingsPageContainer/content/notifications/notifications.jsx

```diff
@@ -100,11 +100,11 @@ export const Notifications = () => {
       </Layout>
       {allNotificationPlugins.map((item) => (
         <RuleGroup
+          ruleFields={item.details.ruleFields}
           key={`rule-section-${item.name}`}
           pluginName={item.name}
           isPluginEnabled={item.enabled}
-          typedRules={notificationRulesByTypes[item.name]}
-          notifications={notifications}
+          typedRules={notificationRulesByTypes[item.name] || []}
```

Comment:
I suppose for `RuleGroup` is it not required to know is there a typed rules or not.
I suggest to rename it to just `rules`.